cli/cli.go

```diff
@@ -67,6 +70,21 @@ func (c *App) Parse() error {
 		c.Command = "start"
 	}
 
+	err := c.Validate()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (c App) Validate() error {
```

Comment:
You will be able to use `.Required` if you change the receiver to be a pointer.

```suggestion
func (c *App) Validate() error {
```

Reply:
It give me error  `only a pointer to a struct can be validated`